modules/bundles/spleen_segmentation/configs/metadata.json

```diff
@@ -0,0 +1,76 @@
+{
+    "schema": "https://github.com/Project-MONAI/MONAI-extra-test-data/releases/download/0.8.1/meta_schema_202203171008.json",
+    "version": "0.1.0",
+    "changelog": {
+        "0.1.0": "complete the model package",
+        "0.0.1": "initialize the model package structure"
+    },
+    "monai_version": "0.8.0",
+    "pytorch_version": "1.10.0",
+    "numpy_version": "1.21.2",
+    "optional_packages_version": {
+        "nibabel": "3.2.1"
+    },
+    "task": "Decathlon spleen segmentation",
+    "description": "A pre-trained model for volumetric (3D) segmentation of the spleen from CT image",
+    "authors": "MONAI team",
```

Comment:
how to support this metadata? https://citation-file-format.github.io/about/

Reply:
CC @ericspod for the metadata format discussion.

Thanks.

Reply:
`CITATION.cff` would be another file in the bundle that can be optionally added and referred to a `citation` value in the metadata?